app/models/mandataris.js

```diff
@@ -1,7 +1,7 @@
 import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 
 export default class MandatarisModel extends Model {
-  @attr('language-string') rangorde;
+  @attr('string') rangorde;
```

Comment:
The `string` transform isn't needed if the backend already sends it like that.

```suggestion
  @attr rangorde;
```